docker_services_cli/config.py

```diff
@@ -28,50 +28,100 @@
 DOCKER_SERVICES_FILEPATH = "docker_services_cli/docker-services.yml"
 """Docker services file default path."""
 
-DEFAULT_VERSIONS = {
-    "ES_6_LATEST": "6.8.12",
-    "ES_7_LATEST": "7.9.0",
-    "POSTGRESQL_9_LATEST": "9.6.19",
-    "POSTGRESQL_10_LATEST": "10.14",
-    "POSTGRESQL_11_LATEST": "11.9",
-    "POSTGRESQL_12_LATEST": "12.4",
-    "POSTGRESQL_13_LATEST": "13.0",
-    "MYSQL_5_LATEST": "5.7.31",
-    "MYSQL_8_LATEST": "8.0.21",
-    "REDIS_6_LATEST": "6.0.6",
-    "MEMCACHED_LATEST": "1.6.6",
-    "RABBITMQ_3_LATEST": "3.8.7",
-}
-"""Services default latest versions."""
-
 # Elasticsearch
 ELASTICSEARCH = {
-    "ES_VERSION": "ES_7_LATEST"
+    "ELASTICSEARCH_VERSION": "ELASTICSEARCH_7_LATEST",
+    "DEFAULT_MINOR_VERSIONS": {
+        "ELASTICSEARCH_6_LATEST": "6.8.12",
+        "ELASTICSEARCH_7_LATEST": "7.9.0",
+    },
 }
 """Elasticsearch service configuration."""
 
 # PostrgreSQL
 POSTGRESQL = {
     "POSTGRESQL_VERSION": "POSTGRESQL_9_LATEST",
-    "POSTGRESQL_USER": "invenio",
-    "POSTGRESQL_PASSWORD": "invenio",
-    "POSTGRESQL_DB": "invenio",
+    "DEFAULT_MINOR_VERSIONS": {
+        "POSTGRESQL_9_LATEST": "9.6.19",
+        "POSTGRESQL_10_LATEST": "10.14",
+        "POSTGRESQL_11_LATEST": "11.9",
+        "POSTGRESQL_12_LATEST": "12.4",
+        "POSTGRESQL_13_LATEST": "13.0",
+    },
+    "CONTAINER_CONFIG_ENVIRONMENT_VARIABLES": {
+        "POSTGRESQL_USER": "johndoe",
```

Comment:
minor: you are probably aware but there are many repos that are already configured with `invenio:invenio` on the GA CI. This will affect those.

Reply:
This will definitely break the current GA CI builds because they have `SQLALCHEMY_DATABASE_URI` hardcoded. However, that is actually something we want to fix with this PR because now `SQLALCHEMY_DATABASE_URI` will be controlled by `docker-services-cli` instead of being written in every Invenio repo's GA CI `tests.yml`.

My guiding principle was that I heard we wanted to keep `docker-services-cli` generic, without Invenio dependencies. Setting name, password and db name to `invenio` seems to go against that principle.

Said that, we can leave it like it is, and once we merge and release this tool, we can progressively remove `SQLALCHEMY_DATABASE_URI` from everywhere, and only then come and change this, what do you think?

Reply:
For me is fine, it was just a "heads up" :) Change is good!

Reply:
Let's close this thread, we can leave this for the future, I've left `invenio` for now. If we adopt this in REANA at some point we will make a PR and by this time probably all Invenio's `SQLALCHEMY_DATABASE_URI ` in CI code will be removed.